news/_posts/2019-10-14-prestashop-1.7-modules-latest-releases-and-updates.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title:  "PrestaShop 1.7 modules - Latest releases & updates"
+subtitle: "Get some great news about PrestaShop 1.7 modules!"
+date:   2019-10-14 11:00:00
+authors: [ ColineGinestet ]
+icon: icon-compass
+tags: [1.7, architecture]
+---
+
+All product and core teams have been quite busy over the past 6 months, working hard on delivering PrestaShop 1.7.6, but not only! We have also worked on bringing new features and improving the 1.7 version through our PrestaShop modules. 
+
+The PrestaShop software is usually seen as a single “brick” and users tend to forget that it is actually accompanied by more than 80 built-in modules, which bring many features to your store such as design, marketing and statistics tools. A document listing all of these modules is currently in preparation, we’ll keep you posted when it is ready!
+
+
+## What is a built-in module? 
+
+This is a module developed by the PrestaShop company and community contributors and available by default (either directly in the zip or pushed through our API) on stores. The most important ones are already installed and available in Module Manager, while others can be found in the Module Catalog list and installed in just one click. Some of them do not need any single configuration, but most of them can be configured and edited according to your needs.
+
+These modules are included by default as they are considered necessary features, but of course, if needed, they can be disabled or uninstalled like any other modules.
+
+Bilal already told you about this continuous work last year ([read here](http://build.prestashop.com/news/native-modules-and-smart-menu)) and this article aims at keeping you updated with recent new modules, refactorizations, bug fixes, feature additions, and optimizations, but also to give you some news about what’s coming in the next few months!
+
+
+## So what’s new with PrestaShop modules? 
+
+As you may already know (I hope so!), the new PrestaShop 1.7.6 version has brought some new and improved important modules: 
+
+### Product Comments (current version : 4.0.0)
+
+This new module, which was available for 1.6 has been completely reworked and offers new features for 1.7.6+ stores, including:
```

Comment:
Maybe provide the github release link https://github.com/PrestaShop/productcomments/releases/tag/v4.0.0 ?

Reply:
yes i agree !